linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_offload_logs/tests/correct.pass.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+touch /etc/cron.weekly/audit-offload
```

Comment:
please make sure that the directory exists first. In the tests there is something:
```
+ touch /etc/cron.weekly/audit-offload
touch: cannot touch '/etc/cron.weekly/audit-offload': No such file or directory
```

```suggestion
mkdir -p /etc/cron.weekly/
touch /etc/cron.weekly/audit-offload
```

Reply:
Thanks, I've fixed it.